packages/babel-core/src/gensync-utils/forward.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import gensync, { type Handler } from "gensync";
+
+type MaybePromise<T> = T | Promise<T>;
+
+export default function forward<ActionArgs: mixed[], ActionReturn, Return>(
```

Comment:
It would be good to have some comments in these, since I don't think it's very clear what their expected usecases would be.